truffle.js

```diff
@@ -1,7 +1,16 @@
+/* eslint-disable global-require */
 require("babel-register");
 require("babel-polyfill");
 const TestRPC = require("ethereumjs-testrpc");
 
+const providerUrl = "http://localhost:8545";
+const nanoPath = "44'/60'/0'/0`";
+
+const nanoProvider = () =>
+  process.argv.filter(arg => arg.includes("nano")).length > 0
```

Comment:
add another parameter to nanoProvider = (...., networkName) and filter exact arg => arg === networkName so we can instantiate just one reacting to network name that we are requesting

Reply:
I had an idea similar to _live we can create _nano and this should check if _nano was included this.
Very similar to what is happening in `config.js`